### ISSUE TITLE
nextjs.org/learn chapter 15 code error

### ISSUE THREAD

**Issue by FG-ABC** [vercel/next.js#67731]
### What is the update you wish to see?

Add the submit type to the signout button found in /ui/dashboard/sidenav.tsx

### Is there any context that might help us understand?

In the final part of the 15th chapter of Learn Next.js, the learner is prompted to add the signout action to the form that's dedicated to signing out the user in the dashboard. Though this is the action, the sign out button itself is left without a submit type so the user won't be logged out.

### Does the docs page already exist? Please link to it.

https://nextjs.org/learn/dashboard-app/adding-authentication#updating-the-login-form

**Comment by ashishnd** [comment]
Hey @FG-ABC! I would like to try to solve this issue. Can you please assign it to me?

**Comment by cjy1998** [comment]
Yes, I have found this problem too, and it can be deadly for beginners
